Sources/SentrySwiftUI/SentryPerformanceView.swift

```diff
@@ -0,0 +1,78 @@
+import Foundation
+import Sentry
+import SwiftUI
+#if SWIFT_PACKAGE
+import SentryInternal
+#endif
+
+/// A control to measure performance of you views and send the result as transaction to Sentry.io.
+///
+/// You create a transaction by wrapping your views with this.
+///
+///     SentryPerformanceView {
+///         VStack {
+///             //The part of your content you want to measure
+///         }
+///     }
+///
+/// By default the transaction name will be the first root view, in the case above `VStack`,
+/// you can give your transaction a custom name by providing the name parameter.
+///
+///     SentryPerformanceView("My Awesome Screen") {
+///         VStack {
+///             //The part of your content you want to measure
+///         }
+///     }
+///
+/// Alternatively you can use the view extension:
+///
+///     VStack {
+///         //The part of your content you want to measure
+///     }.sentryTransaction("My Awesome Screen")
```

Comment:
`h`: I'm not 100% happy with the name cause when you use nesting, this doesn't turn into a transaction. Maybe `sentryPerformance` is a better name?